bcdi/postprocessing/facet_analysis.py

```diff
@@ -1347,8 +1347,8 @@ def to_hdf5(self, path_to_data: str) -> None:
         # Save attributes
         with h5py.File(path_to_data, mode="a") as f:
             try:
-                facets = f.create_group("/data/facets")
-
+                facets = f.create_group("entry_1/process_4/")
```

Comment:
stupid question, why this choice of path "entry_1/process_4" ?

Reply:
In the same file there should be an entry_1 (diffraction data); entry_2 (real space data and phase retrieval), and entry_3 postprocessing output (strain)

Reply:
Is it written somewhere in the h5 file that this entry corresponds to the facet recognition (something like "entry_1/process_4/name" = "facet_recognition" for example). I am fine with standard naming at the condition that things remain understandable :)